Documentation/ceph-pool-crd.md

```diff
@@ -127,6 +127,7 @@ See the official rbd mirror documentation on [how to add a bootstrap peer](https
     If erasure coding is used, the data and coding chunks are spread across the configured failure domain.
 
     > **NOTE**: Neither Rook, nor Ceph, prevent the creation of a cluster where the replicated data (or Erasure Coded chunks) can be written safely. By design, Ceph will delay checking for suitable OSDs until a write request is made and this write can hang if there are not sufficient OSDs to satisfy the request.
+* `replicasPerFailureDomain`: Sets up the number of replica to place in a given failure domain.
```

Comment:
Could we add the more detailed description about how to calculate this value here or in the documentation with the example mentioned in the commit message?